backend/chess/models/pieces/king.py

```diff
@@ -62,9 +58,10 @@ def can_castle(self, turn: int, king_pos: Cell, rook_pos: Cell, board: 'AoWBoard
         if not board.is_path_empty(king_pos, rook_pos, turn):
             return False
 
-        # check if rook has moved
-        if not board.get_piece(rook_pos, turn).has_moved():

```

Comment:
werkte dit nou. zo ja commented code.

Reply:
Das dus het leuke he, met die code werkt het dus alleen op local en niet op de website. Ik heb geen idee waarom maar k denk laat het maar staan anders vergeten we die ooit terug te gooien. Zal er even een TOD bijzettne